lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
```

Comment:
I know that `puck` handles this [differently](https://github.com/iconara/puck/blob/master/lib/puck/dependency_resolver.rb#L49), and perhaps it would be possible to borrow that?

I saw that the tests for Ruby 2.1.5 failed on Travis, and I'm thinking that maybe 2.1.5 uses a different version of Bundler where the monkey-patch doesn't work anymore? If so I think it would be preferable to go with the solution that `puck` uses.